acceptance/bundle/integration_whl/interactive_cluster/output.txt

```diff
@@ -47,6 +47,24 @@ Hello from my func
 Got arguments:
 ['my_test_code', 'one', 'two']
 
+=== Make a change to code without version change and run the job again
+>>> [CLI] bundle deploy
+Building python_artifact...
+Uploading my_test_code-0.0.1-py3-none-any.whl...
+Uploading bundle files to /Workspace/Users/[USERNAME]/.bundle/[UNIQUE_NAME]/files...
+Deploying resources...
+Updating deployment state...
+Deployment complete!
+
+>>> [CLI] bundle run some_other_job
+Run URL: [DATABRICKS_URL]/?o=[NUMID]#job/[NUMID]/run/[NUMID]
+
+[TIMESTAMP] "[default] Test Wheel Job [UNIQUE_NAME]" RUNNING 
+[TIMESTAMP] "[default] Test Wheel Job [UNIQUE_NAME]" TERMINATED SUCCESS 
+Hello from my func
```

Comment:
Is it expected for the wheel artifact to not be updated here?

Reply:
Yes, it's a known issue that same version fails to update the code sometimes. That's why we have this https://github.com/databricks/cli/pull/1034 in default-python template.